extensions/mapstore/src/test/java/com/hazelcast/mapstore/AllTypesGenericMapStoreTest.java

```diff
@@ -61,65 +66,82 @@ public class AllTypesGenericMapStoreTest extends JdbcSqlTestSupport {
     public String value;
 
     @Parameterized.Parameter(2)
-    public BiConsumer<GenericRecord, String> verify;
+    public Object singleColumn;
 
     @Parameterized.Parameter(3)
-    public BiConsumer<GenericRecordBuilder, String> setField;
+    public BiConsumer<GenericRecord, String> verify;
+
+    @Parameterized.Parameter(4)
+    public Consumer<Tuple3<GenericRecordBuilder, String, Object>> setField;
```

Comment:
For this I'd create a own functional interface + tuple record (java 17 would be awesome here :P )